filters/apiusagemonitoring/spec.go

```diff
@@ -169,6 +183,23 @@ func (s *apiUsageMonitoringSpec) buildUnknownPathInfo(paths []*pathInfo) *pathIn
 	return s.unknownPath
 }
 
+func (s *apiUsageMonitoringSpec) configInfo(apis []*apiConfig) apiConfigContext {
```

Comment:
This can be a static function (not coupled to the `apiUsageMonitoringSpec` struct).